app/controllers/api_controller/providers.rb

```diff
@@ -5,9 +5,14 @@ module Providers
     CREDENTIALS_ATTR  = "credentials"
     AUTH_TYPE_ATTR    = "auth_type"
     DEFAULT_AUTH_TYPE = "default"
-    ENDPOINT_ATTRS    = %w(hostname ipaddress port)
+    ENDPOINT_ATTRS    = %w(hostname ipaddress port connection_configurations_array).freeze
```

Comment:
@abellotti its a good question :-)
`connection_configurations_array=` is an alias of `connection_configurations=` so POST is the same
the different is when you do GET, 
`connection_configurations_array` will return the same struct as expected by `connection_configurations_array=`
while `connection_configurations` will return a different struct :-( 
I wanted to make the GET and the POST symmetrical, and `connection_configurations` does not do the job.

@abellotti It will be easy to fix this, but i think its better this way, is it ok to use `connection_configurations_array` ?